packager/media/base/container_names.cc

```diff
@@ -1726,41 +1727,44 @@ MediaContainerName DetermineContainer(const uint8_t* buffer, int buffer_size) {
 
 MediaContainerName DetermineContainerFromFormatName(
     const std::string& format_name) {
-  if (base::EqualsCaseInsensitiveASCII(format_name, "aac") ||
-      base::EqualsCaseInsensitiveASCII(format_name, "adts")) {
+  std::string normalized_format_name = format_name;
+  std::transform(format_name.begin(), format_name.end(),
+                 normalized_format_name.begin(), ::tolower);
+
+  if (normalized_format_name == "aac" || normalized_format_name == "adts") {
```

Comment:
Now that this is no longer using a custom comparison method, couldn't it now be a `switch` statement?